travel_platform/travel_app/views.py

```diff
@@ -1,17 +1,19 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+import json
 import requests
 import sys
 
-from django.http import JsonResponse
+from config_parser.config_reader import get_setting
+from django.http import JsonResponse, HttpResponse
 from django.conf import settings
 from django.shortcuts import render
 from django.views.decorators.cache import cache_page
 
 from api.izi_travel.izi_travel_data import *
-from config.conf import CONF_PATH
-
+from config.conf_open_maps import CONF_PATH_OM
+from api.foursquare.foursquare_data import *
 
 # FIXME RomanPryima: remade sys path in a properly way
```

Comment:
@sasha1152, well, config parser should be added into PYTHONPATH after installation from setup.py. What about izi_travel, foursquare API should be independent. It can't depends on izi travel.